core/domain/user_services.py

```diff
@@ -249,7 +309,27 @@ def fetch_gravatar(email):
     return DEFAULT_IDENTICON_DATA_URL
 
 
-def get_user_settings(user_id, strict=False):
+@overload
+def get_user_settings(
+    user_id: str
+) -> user_domain.UserSettings: ...
+
+
+@overload
+def get_user_settings(
+    user_id: str, *, strict: Literal[True]
+) -> user_domain.UserSettings: ...
+
+
+@overload
+def get_user_settings(
+    user_id: str, *, strict: Literal[False]
+) -> Optional[user_domain.UserSettings]: ...
+
+
+def get_user_settings(
+    user_id: str, strict: bool = True
```

Comment:
The docstring below is not updated with the new default value for 'strict'.

Reply:
Updated!